src/containers/StudyManage/MemberControl/MemberControl.tsx

```diff
@@ -1,7 +1,12 @@
+import { useDispatch } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { AxiosError, AxiosResponse } from 'axios';
+import { openAlert } from '@store/slices/flashAlert';
 import {
   studyManagestudyApplicantsType,
```

Comment:
```suggestion
  studyManageStudyApplicantsType,
```
camel case 오탈자 있습니다!